app/features/receive/receive-cashu.tsx

```diff
@@ -147,17 +109,12 @@ export default function ReceiveCashu({ amount, account }: Props) {
 
         <Carousel opts={{ align: 'center', loop: true }}>
           <CarouselContent>
-            <CashuRequestCarouselItem account={account} amount={amount} />
```

Comment:
if we are removing this does it make sense to leave it carousel at all? could be just regular div right?